crates/pathfinder/resources/block_156000.json

```diff
@@ -0,0 +1 @@
0x134692b230b9e1ffa39098904722134159652b09c5bc41d88d6698779d228ff"], "data": ["0x4a0650479aed6c7d0746dfbecdba2b0a65990b17965effc26730b2140c5f961", "0x4c0139b643f42b32711a7fa870cb66ff15396181e6ff1693e7dc5390f8e73a1", "0x1158e460913d00000", "0x0"]}, {"from_address": "0x6262afee0366ca716347bde824f1a6a81937f59be15ae261ea0b76018ba9c6c", "keys": ["0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"], "data": ["0x4c0139b643f42b32711a7fa870cb66ff15396181e6ff1693e7dc5390f8e73a1", "0x4a0650479aed6c7d0746dfbecdba2b0a65990b17965effc26730b2140c5f961", "0xbb95b60a73fba2bc", "0x0"]}, {"from_address": "0x3bbfc5f909187acae55466ccd89b60fdbefa7f9e2cc6c36476b77c522548175", "keys": ["0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"], "data": ["0x4a0650479aed6c7d0746dfbecdba2b0a65990b17965effc26730b2140c5f961", "0x4c0139b643f42b32711a7fa870cb66ff15396181e6ff1693e7dc5390f8e73a1", "0x1158e460913d00000", "0x0"]}, {"from_address": "0x4c0139b643f42b32711a7fa870cb66ff15396181e6ff1693e7dc5390f8e73a1", "keys": ["0x15543c3708653cda9d418b4ccd3be11368e40636c10c44b18cfe756b6d88b29"], "data": ["0x4a0650479aed6c7d0746dfbecdba2b0a65990b17965effc26730b2140c5f961", "0x4a0650479aed6c7d0746dfbecdba2b0a65990b17965effc26730b2140c5f961", "0x3bbfc5f909187acae55466ccd89b60fdbefa7f9e2cc6c36476b77c522548175", "0x6262afee0366ca716347bde824f1a6a81937f59be15ae261ea0b76018ba9c6c", "0x1158e460913d00000", "0x0", "0xbb95b60a73fba2bc", "0x0"]}, {"from_address": "0x6262afee0366ca716347bde824f1a6a81937f59be15ae261ea0b76018ba9c6c", "keys": ["0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"], "data": ["0x4a0650479aed6c7d0746dfbecdba2b0a65990b17965effc26730b2140c5f961", "0xdb2742af70e2dc77badfe8569b7dd334a696dfd9c5cd2e7b07a9e06f695ec1", "0xbb95b60a73fba2bc", "0x0"]}, {"from_address": "0xdb2742af70e2dc77badfe8569b7dd334a696dfd9c5cd2e7b07a9e06f695ec1", "keys": ["0x5ad857f66a5b55f1301ff1ed7e098ac6d4433148f0b72ebc4a2945ab85ad53"], "data": ["0x1737d9fbf8503498d50d00b1b7552a0bbed8157d4d2d24f988f289a28afa9ba", "0x3", "0x1", "0xbb95b60a73fba2bc", "0x0"]}], "execution_resources": {"n_steps": 6532, "builtin_instance_counter": {"pedersen_builtin": 63, "range_check_builtin": 277, "ecdsa_builtin": 1, "output_builtin": 0, "bitwise_builtin": 0, "ec_op_builtin": 0}, "n_memory_holes": 277}, "actual_fee": "0x2fe202739d3a"}, {"transaction_index": 28, "transaction_hash": "0x60227db1018fa3ba153db82be9e6e2a7a9c1bef311864fcfeb6ec11ab140139", "l2_to_l1_messages": [], "events": [{"from_address": "0x304356ca76a3f320636f1526c613d470bf0d2266aa1a22326ce762d03a23982", "keys": ["0x5ad857f66a5b55f1301ff1ed7e098ac6d4433148f0b72ebc4a2945ab85ad53"], "data": ["0x7c6cf8008c696e4d07bf979b0b1094621c18ed57b378677c9e7879b85255c10", "0x0"]}], "execution_resources": {"n_steps": 2244, "builtin_instance_counter": {"pedersen_builtin": 96, "range_check_builtin": 41, "ecdsa_builtin": 1, "output_builtin": 0, "bitwise_builtin": 0, "ec_op_builtin": 0}, "n_memory_holes": 162}, "actual_fee": "0x17a99e710f06"}]}
```

Comment:
Fixtures seem to be missing a newline in the end.

Also I think maybe `crates/pathfinder/fixtures` might be more applicable place...?

Reply:
Yes, I can just move these to `fixtures`.

Missing newline: this files are _exactly_ what has been returned by the API, so if the API's not returning a newline it's not there.

Reply:
Moved to the new structure Chris has already committed to main in the meantime.